src/client/app/containers/app.ts

```diff
@@ -144,7 +145,7 @@ export class AppComponent {
         this._store.dispatch(new UI.ToggleImportAction(true));
     }
 
-    shareGist(isPublic: boolean) {
+    shareGist(isPublic: boolean, update: boolean) {
```

Comment:
The project should follow and document a uniform naming convention. It looks like booleans should be prefixed with is* for this function.